### ISSUE TITLE
Support for Crypto & Auth for wayvnc

### ISSUE THREAD

**Issue by Mederim** [gujjwal00/avnc#57]
I tried to connect [wayvnc](https://github.com/any1/wayvnc) with auth enable and it cant connect with wayvnc but other clients like bvnc can do. Hera a list of compatibility clients with [wayvnc/neatvnc ](https://github.com/any1/neatvnc#client-compatibility), and you can see how auth works in this vnc server in [here](https://github.com/any1/wayvnc#encryption--authentication)

**Comment by gujjwal00** [comment]
Please see #54

**Comment by Mederim** [comment]
Yeah it worked without auth, is just a suggestion to enable this security type by wayvnc.
